packages/blocks/src/code-block/styles.ts

```diff
@@ -10,11 +10,11 @@ export const codeBlockStyles = css`
     font-size: var(--affine-font-sm);
     line-height: var(--affine-line-height);
     position: relative;
-    padding: 32px 0px 12px 0px;
+    padding: 32px 24px;
     background: var(--affine-background-code-block);
     border-radius: 10px;
-    margin-top: 24px;
-    margin-bottom: 24px;
+    margin: 24px;
```

Comment:
The padding and margin here should be aligned with design. I think there is no need to set value to margin-left and right.

Reply:
The code block should be look like

![image](https://github.com/toeverything/blocksuite/assets/99816898/e13ede37-bad1-463c-9347-dfc5ccb2e02c)

Could you please give a figma link which contains design the image above show. cc @Svaney-ssman